### ISSUE TITLE
Blog homepage and child pages missing lang attribute in top level <html> tag

### ISSUE THREAD

**Issue by aaronmauchley-adobe** [adobe/business-website#174]
Per WCAG 3.1.1 (level A) compliance the language of the page should be identified.  

All of the blog pages under business.adobe.com/blog/... and the homepage for the blog are missing the "lang" attribute in the top-level <HTML> tag of the page(s).  This should be added to be compliant.  See the report information as shown in the screenshots from both Axe and WebAIM.org's WAVE Tool.

![Screen Shot 2021-12-03 at 11 26 23 AM](https://user-images.githubusercontent.com/56042473/144654787-e0dee855-236e-4029-bbf1-31e2a13ad02c.png)

![Screen Shot 2021-12-03 at 11 26 00 AM](https://user-images.githubusercontent.com/56042473/144654792-43ad5794-ab7b-466f-a4ef-54d72ed755a9.png)
